poker-pals/src/components/Login.vue

```diff
@@ -104,8 +104,17 @@
       });
 
       // Navigate to tables page if authentication was successful
-      this.socket.on("authenticated", userID => {
-        this.$router.push({ name: "Tables", params: {authenticated: true, userID: userID}});
+      this.socket.on("authenticatedUser", userID => {
+        this.$router.replace({ name: "Tables", params: {authenticated: true, socket: this.socket, userID: userID}});
+      });
+
+      // Navigate to reports page if authentication was successful
+      this.socket.on("authenticatedAdmin", userID => {
```

Comment:
I've done this as part of my ticket. Can we remove admin vs user changes from this PR? My branch is closely coupled with the logic I've added for that

Reply:
ive removed all the admin code